examples/todos-app/pnpm-lock.yaml

```diff
@@ -0,0 +1,4749 @@
+lockfileVersion: 5.3
```

Comment:
sure, no problem. ok if I just add it to the .gitignore (as I'm using the **pnpm** pkg manager and this file will just accidentally come back otherwise)?

Reply:
Yeah the top level .gitignore already includes npm and yarn lockfiles so it should be fine imo.